src/frontend/src/lib/types/data.context.ts

```diff
@@ -1,8 +1,14 @@
 import type { Writable } from 'svelte/store';
 
+export enum DataStoreStateEnum {
```

Comment:
"state" sound a bit generic, do you have another idea of term? Maybe "action" or "form"?